jobs/pacman-data-shipper/pom.xml

```diff
@@ -221,9 +221,8 @@
 						<configuration>
 							<target>
 								<copy file="target/data-shipper-jar-with-dependencies.jar" tofile="../../dist/jobs/data-shipper-jar-with-dependencies.jar"/>
-								<copy file="target/data-shipper-jar-with-dependencies.jar" tofile="../../dist/jobs/data-shipper-azure.jar"/>
-								<copy file="target/data-shipper-jar-with-dependencies.jar" tofile="../../dist/jobs/data-shipper-gcp.jar"/>
-								<copy file="target/data-shipper-jar-with-dependencies.jar" tofile="../../dist/jobs/data-shipper-redhat.jar"/>
+								<copy file="target/data-shipper-jar-with-dependencies.jar" tofile="../../dist/jobs/data-shipper.jar"/>
+
```

Comment:
Redundant empty line

Reply:
Removed these lines in latest push
@santhosh-challa as per your review comments removed unused jar from pom.xml and formatted the code and removed empty line